### PR TITLE
build: Publish APIC and Engine multi-arch images

### PR DESCRIPTION
**Pull request by gbouv**
## Description:
API container and Engine Docker images are now published for both Mac M1 and Linux AMD64 architecture

## Is this change user facing?
YES

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -849,21 +849,19 @@ jobs:
       - run: echo "${DOCKER_PASSWORD}" | docker login -u ${DOCKER_USERNAME} --password-stdin
 
       - run: |
-          core/server/scripts/build.sh
```
Comment (h4ck3rk3y) [inline .circleci/config.yml:852]: nit: can we put tiny comment on top of what "true" is

File: core/server/scripts/build.sh [hunk 2]
```diff
@@ -12,6 +12,8 @@ source "${script_dirpath}/_constants.env"
 
 BUILD_DIRNAME="build"
 
+DEFAULT_SKIP_DOCKER_IMAGE_BUILDING=false
```
Comment (h4ck3rk3y) [inline core/server/scripts/build.sh:15]: super nit: how do you compare this to DEFAULT_DOCKER_BUILD_IMAGE = false; and then you pass in true above

**Review by h4ck3rk3y — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (gbouv) [reply .circleci/config.yml:852]: Sure, I'll store the `true` in a variable with an explicit name 👍

File: core/server/scripts/build.sh (hunk 2 above)
Comment (gbouv) [reply core/server/scripts/build.sh:15]: So, those are different, on docker_image_builder we have `push_to_registry_container`, and on build.sh this is `skip_docker_image_building`. So, slightly different. I could switch `skip_docker_image_building` to `build_docker_image` and turn the default to true, no strong opinion here 🤷

**Pull request merged**